src/Commands/ProjectCreateCommand.php

```diff
@@ -139,6 +140,66 @@ public function validateCreateProject(CommandData $commandData)
         $this->providerManager()->validateCredentials();
     }
 
+    /**
+     * Get CI short name based on the CI provider.
+     */
+    public function getCiShortName($ci_provider) {
+        $reflect = new \ReflectionClass($ci_provider);
+        switch ($reflect->getShortName()) {
+            case 'CircleCIProvider':
+                return 'circleci';
+
+            case 'GitLabCIProvider':
+                return 'gitlabci';
+
+            case 'BitbucketPipelinesProvider':
+                return 'bitbucket';
+
+            case 'GithubActionsProvider':
+                return 'github';
+        }
+    }
+
+    /**
+     * Copy CI files from the given/default repo.
+     */
+    public function copyCiFiles($ci_provider, $created_folder, $cms_version = 'd8', $repo = '') {
+        $short_name = $this->getCiShortName($ci_provider);
+        if (!$repo) {
+            $repo = 'git@github.com:kporras07/tbt-ci-integrations.git';
```

Comment:
I'll probably move this to default value unless some special logic is needed.

@greg-1-anderson should we put this repo now under pantheon-systems?

Reply:
Yeah, do you have access rights to move it?

Reply:
I could transfer the repository and someone with enough access on pantheon-systems should accept it. Should I rename it before using this transfer option? I'm guessing you would be able to accept the transfer, is that right?

Reply:
Done